cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/DynamicInvokeResolver.kt

```diff
@@ -58,6 +58,7 @@ import java.util.function.Consumer
  */
 @DependsOn(SymbolResolver::class)
 @DependsOn(DFGPass::class)
+@DependsOn(ControlFlowSensitiveDFGPass::class, true)
```

Comment:
```suggestion
@DependsOn(ControlFlowSensitiveDFGPass::class, softDependency = true)
```

For better readability, otherwise its hard to know what "true" is